### ISSUE TITLE
Create a project bug - Error messaging saying that my address is being used for another project, but it isn't/shouldn't be

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#4914]
I was editing the Giveth project.
First, I made edits and then published. Then, I realized there was a mistake, so went to make one more edit in the body again (while it was unlisted).
After editing the body, I wasn't able to "publish" because now it claimed that my address giv.eth was being used for another project - but I didn't change the address, so that's impossible.

I think there is an issue happening somehow where if the project is unlisted, our checks on the address are bugging out.

cc @divine-comedian 

![image](https://github.com/user-attachments/assets/da6d2891-b859-4fa8-b1a0-bf2c56ed50f2)

**Comment by divine-comedian** [comment]
@laurenluz if you made the edits, published, listed it in the admin again then made edits again (causing it to become unlisted) then the error above did not occur?

**Comment by kkatusic** [comment]
@laurenluz, cc @divine-comedian, I found one project that is using same address for Arbitrum network: https://giveth.io/project/1-0?tab=donations

**Comment by laurenluz** [comment]
I tried again, and actually, yeah, I have the same issue even if it's listed.
however, I didn't have this issue the FIRST time I pushed the edit. on the 2nd time...

A fix that would help: that project @kkatusic you mentioned is inactive. projects that are inactive should not conflict with active projects if they have the same address.

There are more issues on create a project happening too wrt to this image thing:

For example:
- when I add an image, for no reason at all, it just shows me it blurry. sometimes this doesn't happen with the same image.
- the bounds of the image go beyond the bounds of the editing box

https://github.com/user-attachments/assets/77f46620-0cd2-4b8b-92a5-8cc3d6004534

**Comment by kkatusic** [comment]
@laurenluz I checked image behaviours inside editor, blurred image is showing you that is process of the uploading same in progress, take some time for the larger images.

The bounds go outside like you wrote, we need to check editor update does same behaviour has been fixed.

**Comment by laurenluz** [comment]
@kkatusic I think the best solution for the address issue is the 3rd one: write code that will exclude checking address on project that are inactive.

Projects become inactive when the project owner decides to do it. They may one day want to come back to reactivate it. Likely not, but possible. 

did you already upgrade the image editor?

**Comment by divine-comedian** [comment]
> @kkatusic I think the best solution for the address issue is the 3rd one: write code that will exclude checking address on project that are inactive.
> 
> Projects become inactive when the project owner decides to do it. They may one day want to come back to reactivate it. Likely not, but possible.
> 
> did you already upgrade the image editor?

I don't think this is a good idea, since we will run into issues if/when a project chooses to reactivate

**Comment by divine-comedian** [comment]
> I tried again, and actually, yeah, I have the same issue even if it's listed. however, I didn't have this issue the FIRST time I pushed the edit. on the 2nd time...
> 
> A fix that would help: that project @kkatusic you mentioned is inactive. projects that are inactive should not conflict with active projects if they have the same address.
> 
> There are more issues on create a project happening too wrt to this image thing:
> 
> For example:
> 
> * when I add an image, for no reason at all, it just shows me it blurry. sometimes this doesn't happen with the same image.
> * the bounds of the image go beyond the bounds of the editing box
> 
>  Screen.Recording.2025-01-03.at.5.22.54.PM.mov

I follow up this issue here- > https://github.com/Giveth/giveth-dapps-v2/issues/4925

**Comment by laurenluz** [comment]
> I don't think this is a good idea, since we will run into issues if/when a project chooses to reactivate, this can be done arbitrarily by the user at anytime and we will run into issues later on trying to fix that bug.

We should just plan for that within the flow. When they try to reactivate - if someone else used that address for another project - they could get some message like: "oops, that address is being used by another project. change the recipient address in order to reactivate the project". and ofc, we'd need to allow people to edit inactive projects.

There needs to be some user flow designed for it. removing wallet addresses might confuse people, and removing the project when someone sets it as inactive is not really a good option because as you said, someone might even do this by accident.

**Comment by divine-comedian** [comment]
> > I don't think this is a good idea, since we will run into issues if/when a project chooses to reactivate, this can be done arbitrarily by the user at anytime and we will run into issues later on trying to fix that bug.
> 
> We should just plan for that within the flow. When they try to reactivate - if someone else used that address for another project - they could get some message like: "oops, that address is being used by another project. change the recipient address in order to reactivate the project". and ofc, we'd need to allow people to edit inactive projects.
> 
> If we do anything to address it, there would need to be some user flow designed for it. removing wallet addresses might confuse people, and removing the project when someone sets it as inactive is not really a good option because as you said, someone might even do this by accident.

I think this will create more complexities especially in the back-end where we store the project data. The impact-graph is a very fickle beast these days. 

We noticed that this validation of checking existing recipient addresses should have been happening more consistently and for some reason it only started working recently - even though the code remained unchanged for 3 years... 

The quickest and most direct solution is to edit the project data that is interfering with updating the Giveth project and let's keep a close eye if similar issues are being reported by users.

**Comment by laurenluz** [comment]
sounds good. so @kkatusic maybe just remove the giv.eth address from that (inactive) project...

But worth noting that also, when I created the issue initially... those addresses were used for the giveth project _already_ (I didn't edit them for a long long time) so there was some error in our system prior to me creating the issue that led to the duplication in the first place.

**Comment by divine-comedian** [comment]
> sounds good. so @kkatusic maybe just remove the giv.eth address from that (inactive) project...
> 
> But worth noting that also, when I created the issue initially... those addresses were used for the giveth project _already_ (I didn't edit them for a long long time) so there was some error in our system prior to me creating the issue that led to the duplication in the first place.

Exactly, that's the confusing bit is that some piece of code that hasn't been touched for 3 years is now working, when maybe it wasn't working for... god knows how long

So far I've only heard of this issue from you and Maryjaf cause she has like 3581945197651056914 test projects

**Comment by maryjaf** [comment]
I’m encountering an error on https://giveth.io/project/maryam-test. Could you please apply the same change that you made to the Giveth project to this project as well, so I can test it? @kkatusic

**Comment by kkatusic** [comment]
> I’m encountering an error on https://giveth.io/project/maryam-test. Could you please apply the same change that you made to the Giveth project to this project as well, so I can test it? @kkatusic

@maryjaf done. Please test it when you find time.

**Comment by maryjaf** [comment]
Thanks , it has been fixed for me
